src/jvmMain/kotlin/org/treeWare/metaModel/encoder/DotMetaModelEncoder.kt

```diff
@@ -127,8 +127,11 @@ private fun encodeAssociationField(fieldMeta: EntityModel, dotWriter: DotWriter)
     val entityFullName = getMetaModelResolved(fieldMeta.parent.parent)?.fullName
     val fullName = getMetaModelResolved(resolvedEntity)?.fullName
     val name = getMetaName(fieldMeta)
-
-    dotWriter.linksWriteLine("""  "$entityFullName":"$name:e" -> "$fullName":0 [style="dashed" color=sienna]""")
+    if (entityFullName == fullName) {
```

Comment:
nit: it is hard to tell that the only difference between the two commands is the `:0:n` and `:0`. A separate `val` for it would be better. Assign the val using an if-else expression, and then string-interpolate that val in the string below.

While looking up what to call that `val`, I saw the following docs called it "compass point" and that "ne" (north-east) is an option. Did you try that value? How does that look?